HWIMO-BUILD

```diff
@@ -9,6 +9,12 @@
 set -e
 set -x
 
+# $1 should be a file which contains the release version.
+# For example:  PKG_VERSION=on-http~1.2.3-rc-abc...
+if [ -f "$1" ];then
```

Comment:
I'm still argue why we can't put those into ENV VAR like before..
HWIMO-BUILD never takes explict param before~